vampire.cpp

```diff
@@ -670,9 +678,9 @@ void axiomSelectionMode()
 
   ScopedPtr<Problem> prb(UIHelper::getInputProblem(*env.options));
 
-  if (prb->hasFOOL()) {
+  /*if (prb->hasFOOL()) {
     FOOLElimination().apply(*prb);
-  }
+  }*/
```

Comment:
I never used axiomSelectionMode, but FOOLElimination something to avoid at the moment?